lib/clientSideScripts/scrollToPosition.spec.ts

```diff
@@ -1,27 +0,0 @@
-import scrollToPosition from './scrollToPosition';
-
-const CONFIGURABLE = {
-  writable: true,
-  configurable: true,
-};
-
-describe('scrollToPosition', () => {
```

Comment:
Wouldn't you want to test the new implementation even if it's much simpler?

Reply:
This is default browser implementation and imho not worth testing, it's not risk in my opinion

Reply:
Fair enough, but there is a tiny bit of logic :-)

Reply:
Haha, yeah, there is, but not a big risk